indico/modules/events/papers/forms.py

```diff
@@ -39,8 +39,10 @@ class PaperTeamsForm(IndicoForm):
                                   event=lambda form: form.event,
                                   description=_('List of users allowed to manage the call for papers'))
     judges = PrincipalListField(_('Judges'),
+                                event=lambda form: form.event,
```

Comment:
Why is this needed?

Reply:
Nevermind, without it the PrincipalListField does not have the event id, which is needed in the plugin code in order to build the API URL.